### PR TITLE
Add execution filter to tasklist tasks CLI command

### PR DESCRIPTION
**Pull request by feedmeapples**
**What changed?**
admin CLI command that allows to list Tasks from a TaskList now has an optional Workflow Id and Run Id filter. 
The command itself now utilizes paginate util to switch between table and json views 

**Why?**
Allows to narrow down to specific workflow runs when discovering tasks of a tasklist

**How did you test it?**
ran tests and command

**Potential risks**
low risk

### CODE REVIEW

File: tools/cli/admin.go [hunk 1]
```diff
@@ -725,7 +725,11 @@ func newAdminTaskListCommands() []cli.Command {
 		{
 			Name:  "list_tasks",
 			Usage: "List tasks of a tasklist",
-			Flags: append(getDBFlags(),
+			Flags: append(
+				append(
+					append(
+						getDBFlags(), flagsForExecution...),
```
Comment (alexshtin) [inline tools/cli/admin.go:731]: Nit: shouldn't it be one line?

File: tools/cli/adminTaskListCommands.go [hunk 2]
```diff
@@ -128,9 +130,34 @@ func AdminListTaskListTasks(c *cli.Context) {
...
-		ErrorAndExit("Failed to get Tasks", err)
+	paginationFunc := func(paginationToken []byte) ([]interface{}, []byte, error) {
+		response, err := taskManager.GetTasks(req)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		tasks := response.Tasks
+		if workflowID != "" {
+			all := tasks
```
Comment (alexshtin) [inline tools/cli/adminTaskListCommands.go:141]: You might want to use this: https://github.com/golang/go/wiki/SliceTricks#filter-in-place (read entire page, it is fun and useful).

Comment (alexshtin) [reply tools/cli/adminTaskListCommands.go:141]: or this: https://github.com/golang/go/wiki/SliceTricks#filtering-without-allocating

File: tools/cli/adminTaskListCommands.go [hunk 3]
```diff
@@ -135,9 +137,34 @@ func AdminListTaskListTasks(c *cli.Context) {
...
+
+			for _, task := range all {
+				if task.Data.WorkflowId != workflowID {
+					continue
+				}
+				if runID != "" && task.Data.RunId != runID {
+					continue
+				}
+				tasks = append(tasks, task)
+			}
```
Comment (alexshtin) [inline tools/cli/adminTaskListCommands.go:159]: Isn't this better? 
```suggestion
			filteredTasks := tasks[:0]

			for _, task := range tasks {
				if task.Data.WorkflowId != workflowID {
					continue
				}
				if runID != "" && task.Data.RunId != runID {
					continue
				}
				filteredTasks = append(filteredTasks, task)
			}
			
			tasks = filteredTasks
```
It's up to you though.

Comment (feedmeapples) [reply tools/cli/adminTaskListCommands.go:159]: looks good, i will test with that change

Comment (feedmeapples) [reply tools/cli/adminTaskListCommands.go:159]: sure, looks good

**Pull request merged**